services/management/ia/templates_service.go

```diff
@@ -71,6 +78,34 @@ func NewTemplatesService(db *reform.DB) *TemplatesService {
 	}
 }
 
+// Run creates tempaltes and rules dir
+func (svc *TemplatesService) Run() {
```

Comment:
Run methods should accept context, run in a loop and exit when context is canceled.

It looks looks like this code should be inside `NewTemplatesService`